tests/e2e/e2e_os_test.go

```diff
@@ -24,15 +24,21 @@ func waitForBuildComplete(namespace string, buildName string) error {
 	})
 }
 
-func runKedgeS2i(imageName string, baseImage string) error {
-	s2iCmd := BinaryLocation + " build --s2i --image " + imageName + " -b " + baseImage
+func runKedgeS2i(nameSpace string, baseImage string, contextDir string) error {
+	s2iCmd := BinaryLocation + " build --s2i --image " + nameSpace + "-img -b " + baseImage +
```

Comment:
I think it should be,

```
s2iCmd := BinaryLocation + " build --s2i --image " + baseImage +" -c " + contextDir + " -n " + nameSpace
```

Reply:
Bit confused here, tests works fine locally:

```
 $ make test-e2e-os 
TEST=os make test-e2e 
make[1]: Entering directory '/home/snarwade/go/src/github.com/kedgeproject/kedge'
./scripts/run_e2e.sh
======================================
| Running end-to-end cluster tests.  |
| Tests will be ran against a k8s    |
| cluster in separate namespaces     |
|                                    |
| Use command:                       |
| VERBOSE=yes make test-e2e-os       |
| for verbosity.                     |
======================================

ok  	github.com/kedgeproject/kedge/tests/e2e	246.458s
make[1]: Leaving directory '/home/snarwade/go/src/github.com/kedgeproject/kedge'
```

Reply:
@surajnarwade the `--image` is for the name of the image to be generated. `--baseimage` is for baseimage.